packages/core/src/modules/attendance/repositories/index.ts

```diff
@@ -0,0 +1,26 @@
+import { Kysely } from "kysely"
+import { _AttendancePoolRepository, _PoolRepositoryImpl } from "./attendance-pool-repository"
+import { _AttendanceRepository, _AttendanceRepositoryImpl } from "./attendance-repository"
+import { _AttendeeRepository, _AttendeeRepositoryImpl } from "./attendee-repository"
+import { _WaitlistAttendeRepository, _WaitlistAttendeRepositoryImpl } from "./waitlist-attendee-repository"
+import { Database } from "@dotkomonline/db"
+
+export interface AttendanceRepository {
+  attendance: _AttendanceRepository
+  pool: _AttendancePoolRepository
+  waitlistAttendee: _WaitlistAttendeRepository
+  attendee: _AttendeeRepository
+}
+
+export class AttendanceRepositoryImpl implements AttendanceRepository {
+  attendance: _AttendanceRepository
+  pool: _AttendancePoolRepository
+  waitlistAttendee: _WaitlistAttendeRepository
+  attendee: _AttendeeRepository
+  constructor(db: Kysely<Database>) {
+    this.attendance = new _AttendanceRepositoryImpl(db)
+    this.pool = new _PoolRepositoryImpl(db)
+    this.waitlistAttendee = new _WaitlistAttendeRepositoryImpl(db)
+    this.attendee = new _AttendeeRepositoryImpl(db)
+  }
+}
```

Comment:
I think this should be moved into the file where everything else is instantiated, even though it's not used anywhere. Also remove the _ prefix on all of them